front/lib/resources/file_resource.ts

```diff
@@ -197,6 +197,13 @@ export class FileResource extends BaseResource<FileModel> {
       .createReadStream();
   }
```

Comment:
We want to add a sIdToModelId here as well no?

Reply:
Will add in a follow up PR when I'll start using it.